rootfs-builder/clearlinux/rootfs_lib.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+
+
+
+check_program(){
+	type "$1" >/dev/null 2>&1
+}
+
+generate_dnf_config()
+{
+	cat > "${DNF_CONF}" << EOF
+[main]
+cachedir=/var/cache/dnf/clear/
+keepcache=0
+debuglevel=2
+logfile=/var/log/dnf.log
+exactarch=1
+obsoletes=1
+gpgcheck=0
```

Comment:
We should probably comment this (here and elsewhere) with a warning message.

Reply:
@jodh-intel Added a warning for Clear Linux based rootfs. For Fedora I am still looking for a way to import use local gpg keys.

Reply:
Thanks.